clients/api-proxy/rne/index.ts

```diff
@@ -11,6 +11,6 @@ import { clientAPIProxy } from '../client';
 export const fetchRNEImmatriculation = async (siren: Siren, useCache = true) =>
   clientAPIProxy(
     routes.proxy.rne + siren,
-    { timeout: constants.timeout.XL },
+    { timeout: constants.timeout.XXL },
```

Comment:
XXL = 20s

I think we can create `XXXL` for in-browser calls ! (30 sec ? Maybe more depending on what is the default browser timeout)

Reply:
Actually I am not even sure there is a timeout for ajax calls

Reply:
> Actually I am not even sure there is a timeout for ajax calls

I don't think there is, or it's browser-based. However, the value here is the timeout for the `ProxyAPI` client, which is used in a `/api` route of the next app.

Reply:
👍
I created a dedicated issue 
https://github.com/etalab/annuaire-entreprises-api-proxy/issues/17